pyat/at/acceptance/acceptance.py

```diff
@@ -115,14 +121,23 @@ def get_acceptance(
     if start_method is not None:
         kwargs["start_method"] = start_method
 
+    # For backward compatibility (use_mp can be a boolean)
+    if isinstance(use_mp,bool) and use_mp:
+        use_mp = MPMode.CPU
```

Comment:
And a last one:
```python
if use_mp is True:
    use_mp = MPMode.CPU
```

Different from `if use_mp:` which would be `True` for all except `False`.